azure-pipelines.yml

```diff
@@ -31,11 +31,10 @@ jobs:
                 architecture: 'x64'
 
           - script: |
-                pip install "spacy>=3.2.0,<3.5.0"
+                pip install "spacy>=3.2.0,<3.5.0"  
                 pip install pytest
                 pip install wheel
             displayName: 'Install dependencies'
 
-          - task: Bash@3
-            inputs:
-              filePath: 'run-tests.sh'
\ No newline at end of file
+          - script: bash run-tests.sh python
+            displayName: 'Run tests'
```

Comment:
What is the difference in bash/azure here?

Reply:
None that I noticed. The [Azure bash operator](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/bash-v3?view=azure-pipelines) is neater, but is just running `bash` under the hood. For our purpose just running `bash` directly seemed easier to read to me.